Components/Modal/modal.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./../../css/main.css">
+    <link rel="stylesheet" href="./../../css/components.css">
+    <link rel="stylesheet" href="./../Inputs/input.css">
+    <link rel="stylesheet" href="./../Buttons/button.css">
+    <link rel="stylesheet" href="./../Cards/card.css">
+    <link rel="stylesheet" href="./modal.css">
```

Comment:
Just a suggestion, instead of linking all css files, you can import these files in one file say style.css and link it here.

Reply:
Thanks Roma for suggestion I haven't  hosted the site yet that's why was keeping styles separate but will do the refractor soon and arrange in single file.